### PR TITLE
[Doc] The new setting "httpCookies" can affect on tracking visits on sites where the protocol "http" is used

### PR DESCRIPTION
**Pull request by sc-illiakovalenko**
The new setting "httpCookies" can affect on tracking visits on sites where the protocol "http" is used

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code follows the code style of this project.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

**Comment by nickwesselman** [comment]
Hey @sc-illiakovalenko --

Let's add this to the locations where we talk about analytics instead:
https://jss.sitecore.com/docs/fundamentals/services/tracking#jss-tracking (at the bottom of the first section)
https://jss.sitecore.com/docs/fundamentals/services/layout-service#layout-service-and-analytics

And let's revise the content to this:

> _NOTE_: As of Sitecore 10.0.1, Sitecore sets the `Secure` flag on all cookies by default. This can impact JSS local development in *Connected* and *Headless* modes, as the proxied Sitecore cookies (including analytics cookies) will be rejected by the browser if your application is not running under HTTPS, and thus visits will not be tracked and content may not be personalized. To work around this, you can either:
> * Enable HTTPS in your local environment by modifying the node server, using a local reverse proxy, or a service such as ngrok.
> * Transform the Sitecore Web.config and set `requireSSL` to `false` in the `httpCookies` configuration. **This is not recommended for production.**
>`<httpCookies sameSite="None" requireSSL="false" />`

**Review by nickwesselman — CHANGES_REQUESTED** [review]
See previous comment

**Comment by sc-illiakovalenko** [comment]
@nickwesselman Thank you for review, does it fit now?

File: docs/data/routes/docs/fundamentals/services/layout-service/en.md [hunk 1]
```diff
@@ -87,6 +87,11 @@ The Layout Service executes within the Sitecore MVC rendering engine, and thus r
 
 If you don't want analytics tracking for your JSS app, or for particular Layout Service calls, set the `tracking` parameter to `false`.
 
+> _NOTE_: As of Sitecore 10.0.1, Sitecore sets the `Secure` flag on all cookies by default. This can impact JSS local development in *Connected* and *Headless* modes, as the proxied Sitecore cookies (including analytics cookies) will be rejected by the browser if your application is not running under HTTPS, and thus visits will not be tracked and content may not be personalized. To work around this, you can either:
+> * Enable HTTPS in your local environment by modifying the node server, using a local reverse proxy, or using a service such as ngrok.
+> * If you are running Sitecore in containers for development, you can make use of the Traefik reverse proxy that is provided in the `docker-compose` environment.
```
Comment (nickwesselman) [inline docs/data/routes/docs/fundamentals/services/layout-service/en.md:92]: Can you make sure this bullet is further indented, so it falls under the top bullet? Same in the other page. Thanks!

**Review by nickwesselman — CHANGES_REQUESTED** [review]

File: docs/data/routes/docs/fundamentals/services/layout-service/en.md (hunk 1 above)
Comment (sc-illiakovalenko) [reply docs/data/routes/docs/fundamentals/services/layout-service/en.md]: @nickwesselman Added

**Review by nickwesselman — APPROVED** [review]

**Pull request merged**